README.rst

```diff
@@ -2,56 +2,129 @@
+in the three general constraints.
+
+INH's three types of constraints are:
+
+1) "hard shift" constraint: requires that at least one nurse is assigned for
+   each working day.
+
+2) "hard nurse" constraint: ensures that no nurse works two or more 
+   consecutive days.
+
+3) "soft nurse" constraint: requires that all nurses should have roughly
+   even work schedules.
+
+This demo seeks to obtain reasonable results for a nurse schedule, based on
+INH's model. Our implementation attempts to find a schedule for a number N 
+of nurses and a number D of days that satisfies the following conditions:
+
+* One, and only one, nurse has been assigned to each day (hard shift 
+  constraint)
+* No nurse works two days in a row (hard nurse constraint)
+* The nurses should work the same number of days
+
+Running the demo results in the following output, at the command-line:
+
+    Size  33
+
+    Energy  0.5999999999999694
+
+    Checking Hard shift constraint  0.0
+
+    Checking Hard nurse constraint  0.0
+
+    Checking Soft nurse constraint  0.6
+
+========= =  =  =  =  =  =  =  =  =  =  ==
+   Day    0  1  2  3  4  5  6  7  8  9  10
+========= =  =  =  =  =  =  =  =  =  =  ==
+Nurse  0           X     X     X     X   
+Nurse  1     X        X     X           X
+Nurse  2  X     X                 X      
+========= =  =  =  =  =  =  =  =  =  =  ==
+
+The results show the following:
+
+* One, and only one, nurse has been assigned to each day
+* No nurse works two days in a row
+* Two nurses work 4 days, and one works three days. Because two nurses work
+  one extra day each, the soft nurse constraint is nonzero. Each nurse 
+  working one extra day contributes a total of gamma to the energy. Since
+  gamma is 0.3, the total energy is expected to be 0.6.
 
 Usage
 -----
 
-A simple command that runs the demo. For example,
+To run the demo, run the command
 
 .. code-block:: bash
 
-  python <demo_name>.py
+  python nurse_scheduling.py
 
 
 Code Overview
 -------------
 
-A general overview of how the code works.
+Here is a general overview of the Nurse Scheduling code:
 
-Prefer bite-sized descriptions in bullet points:
-
-* Here's an example bullet point
+* Assign the size of the problem (number of nurses and days) and parameters
+* Compute the "penalty matrix" J
+* Develop the QUBO matrix
+* Run the problem (solve the QUBO)
+* Compute the hard shift constraint sum
+* Compute the hard nurse constraint sum
+* Compute the soft nurse constraint sum
+* Print the nurse schedule
 
+Note that the total of the three constraint sums should equal the energy.
 
 Code Specifics
 --------------
 
-Notable parts of the code implementation.
+Some notes on the code:
 
-This is the place to:
+* We use a vector to store the two-dimensional (nurse, day) matrix:
 
-* Highlight a part of the code implementation
-* Talk about unusual or potentially difficult parts of the code
-* Explain a code decision
+(0, 0) (0, 1) (0, 2)... (0, D) (1, 0) (1, 1)... (1, D)
```

Comment:
I think you need to add two whitespaces in front in order for the list to line up with the bullet point (in the above line).

```
  (0, 0) (0, 1) ..
```

Reply:
Thank you.